gnocchi/rest/aggregates/api.py

```diff
@@ -238,36 +219,35 @@ def post(self, start=None, stop=None, granularity=None,
 
             groupby = sorted(set(api.arg_to_list(groupby)))
             sorts = groupby if groupby else api.RESOURCE_DEFAULT_PAGINATION
-            try:
+            if not groupby:
                 resources = pecan.request.indexer.list_resources(
```

Comment:
Please add this back into a try statement and catch the IndexerException and return 400 as was done before. You can move it inside the try statement below.

Reply:
Yes, sure, I missed that. 
Thanks.